docs/carbon-as-spark-datasource-guide.md

```diff
@@ -15,19 +15,20 @@
     limitations under the License.
 -->
 
-# Carbon as Spark's datasource guide
+# Carbon as Spark's Datasource
 
-Carbon fileformat can be integrated to Spark using datasource to read and write data without using CarbonSession.
+The Carbon fileformat is now integrated as Spark datasource for read and write operation without using CarbonSession. This is useful for the users who wants same Spark datasource. 
```

Comment:
Sentence not correct.This is useful for users who wants to use carbondata as a spark's data source